ios/application_swiftui/ContentView.swift

```diff
@@ -1,8 +1,45 @@
 import SwiftUI
 
 struct ContentView: View {
-	var body: some View {
-		Text("Hello, world!")
-			.padding()
-	}
+    @State private var items = [Date]()
+
+    var body: some View {
+        NavigationView {
+            VStack {
+                List {
+                    ForEach(items, id: \.self) { item in
+                        Text(item.description)
+                    }
+                    .onDelete(perform: delete)
+                }.toolbar {
+                    ToolbarItem(placement: .navigationBarLeading) {
+                        EditButton()
+                    }
+                }
+                .listStyle(InsetGroupedListStyle())
+            }
+            .navigationTitle("Content View")
+            .toolbar {
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    Button(action: {
+                        withAnimation(.easeInOut) {
+                            items.insert(Date(), at: 0)
+                        }
+                    }) {
```

Comment:
nit: Use trailing closure syntax

```suggestion
                    Button {
                        withAnimation(.easeInOut) {
                            items.insert(Date(), at: 0)
                        }
                    } label: {
```